src/cli.rs

```diff
@@ -338,6 +338,17 @@ async fn process_line(readline: Result<String, ReadlineError>, ctx: &mut Context
         Ok(line) if line.trim() == "" => LineResult::Success(line.clone()),
 
         Ok(line) => {
+            // Replace ~ with the address of the home directory.
+            let mut line_string = String::from(line);
+            if line.contains("~") {
```

Comment:
Do we need it in the whole string? Are there any scenarios where `~` is not the first char?

Reply:
I _think_ `line` here is the entire line, e.g. `cd ~/foo`

Reply:
@superfunc exactly. It applies to all arguments, e.g. `mv ~/foo ~/bar/`

Reply:
What about `~a_user/`, which should expand to the home dir of user `a_user`.